hyphae/src/main.rs

```diff
@@ -0,0 +1,100 @@
+mod config;
+mod kubo_api;
+mod myceli_api;
+
+use std::ops::Sub;
+
+use anyhow::Result;
+use clap::Parser;
+use config::Config;
+use kubo_api::KuboApi;
+use myceli_api::MyceliApi;
+use std::collections::HashSet;
+use std::thread::sleep;
+use std::time::Duration;
+use tracing::{error, info, Level};
+
+pub const RAW_CID_PREFIX: &str = "bafkrei";
```

Comment:
Are these made up values or Kubo/IPFS ones?  If made up, consider making it easier to scan b/c a single letter difference in a non-word (I think?) is hard to differentiate.